cvmfs/swissknife_lease_json.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_SWISSKNIFE_LEASE_JSON_H_
+#define CVMFS_SWISSKNIFE_LEASE_JSON_H_
+
+#include "swissknife_lease_curl.h"
+
+#include <string>
+
+bool ParseAcquireReply(const CurlBuffer& buffer, std::string* session_token);
```

Comment:
I don't know about this one. I don't see the point of making classes for everything, like in Java.
These functions already encapsulate the needed functionality and there would be no state to maintain in the instance of the class. The same applies to the `swissknife_lease_curl.{h,cc}` files, too.

I can put these functions in namespaces, if you think it brings more clarity...

Ok, for the JSON unit tests.